src/app/app.component.ts

```diff
@@ -1,10 +1,19 @@
 import { Component } from '@angular/core';
+import { NavigationChannelComponent } from './navigation-channel/navigation-channel.component';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.css']
+  styleUrls: ['./app.component.css'],
+  providers: [NavigationChannelComponent]
 })
 export class AppComponent {
   title = 'test-editor-web';
+
+    constructor(private navigationChannel: NavigationChannelComponent) {
+    navigationChannel.navEvent$.subscribe(navEvent => {
+        console.log('Received navigation event in app-root:');
+        console.log(navEvent);});
```

Comment:
formatting:
- contents of the constructor should be indented
- `});` should be on a new line